db/migrations/000001_initial.up.sql

```diff
@@ -0,0 +1,108 @@
+DO LANGUAGE plpgsql
+$$
+DECLARE 
+    fw VARCHAR;
+BEGIN
+    SELECT FROM information_schema.tables 
+    INTO fw
+    WHERE  table_schema = 'public' AND  table_name = 'flyway_schema_history';
```

Comment:
This could be improved by checking the last migration was applied. 
If not raise an error to the user to deploy first the release with the last flyway migration.